tests/conftest.py

```diff
@@ -442,6 +442,31 @@ def finalizer():
         Delete the project
         """
         for instance in instances:
+            try:
+                ocp_event = ocp.OCP(kind="Event", namespace=instance.namespace)
+                events = ocp_event.get()
+                event_count = len(events['items'])
+                warn_event_count = 0
+                for event in events['items']:
+                    if event['type'] == "Warning":
+                        warn_event_count += 1
+                log.info(
+                    (
+                        "There were %d events in %s namespace before it's"
+                        " removal (out of which %d were of type Warning)."
+                        " For a full dump of this event list, see DEBUG logs."
+                    ),
+                    event_count,
+                    instance.namespace,
+                    warn_event_count
+                )
+            except Exception as ex:
+                # we don't want any problem to disrupt the teardown itself
+                log.warning(
+                    "failed to get events for %s namespace: %s",
+                    instance.namespace,
+                    ex
```

Comment:
```suggestion
                    "Failed to get events for namespace: %s.\nThe exception is: %s,"
                    instance.namespace,
                    ex
```

Reply:
Wouldn't this produce lines like "exception is exception"?

Reply:
Probably..
So I would change to something like:
```
"Failed to get events for namespace: %s.\n%s," instance.namespace, ex
```

Also, why not using f string?

Reply:
> Also, why not using f string?

Using plain f string for logging breaks [our guidelines](https://ocs-ci.readthedocs.io/en/latest/docs/coding_guidelines.html).

Reply:
Why? I see this:
![Screenshot from 2020-04-27 16-58-48](https://user-images.githubusercontent.com/6973523/80380583-71011100-88a8-11ea-9aa8-cc2f4e4e19c4.png)

Reply:
Read it through till the end.